rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -674,6 +848,8 @@ def load(
         **kwargs: Any,
     ) -> "CountVectorsFeaturizer":
 
+        finetune_mode = kwargs.pop("finetune_mode", False)
+
```

Comment:
This is subject to change based on how https://github.com/RasaHQ/rasa/issues/7329 is fixed.